snippets/cdc-acm-console/cdc-acm-console.conf

```diff
@@ -0,0 +1,9 @@
+CONFIG_USB_DEVICE_STACK=y
+CONFIG_USB_DEVICE_PRODUCT="Zephyr USB console sample"
+CONFIG_USB_DEVICE_PID=0x0004
+CONFIG_USB_DEVICE_INITIALIZE_AT_BOOT=n
```

Comment:
> This is both yes and no - for basic samples, this needs to be y (e.g. hello_world) but for devices that set USB as part of the application it needs to be n. Thingy53 configuration:

Is the thingy53 configuration universal? If so, that widens the scope a bit for the initial implementation to include real kconfig files instead of just .conf files, but I can add that in.